api/src/main/java/com/capitalone/dashboard/service/FeatureServiceImpl.java

```diff
@@ -311,7 +311,7 @@ public FeatureServiceImpl(ComponentRepository componentRepository,
 
 		default:
 			storyEstimates = new ArrayList<>();
-
+			break;
```

Comment:
see screenshot above, PMD complains in eclipse missing break, adding it is harmless and lest us see real issues in PMD.